### ISSUE TITLE
graphql-faker docker certificate has expired

### ISSUE THREAD

**Issue by Antiarchitect** [APIs-guru/graphql-faker#117]
I'm using official docker image of graphql-faker apisguru/graphql-faker:v2.0.0-rc.19 and getting this:
```
Default file ./schema_extension.faker.graphql is used. Specify [file] parameter to change.
Error: Can't get introspection from https://api-voronkov-rename-cache-folders.stages.xxx/graphql:
request to https://api-voronkov-rename-cache-folders.stages.xxx/graphql failed, reason: certificate has expired
    at graphqlRequest.then.catch (/usr/local/share/.config/yarn/global/node_modules/graphql-faker/dist/utils.js:31:15)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
```
But from the local machine all is ok:
```
graphql-faker --extend https://api-voronkov-rename-cache-folders.stages.xxx/graphql
Default file ./schema_extension.faker.graphql is used. Specify [file] parameter to change.

✔ Your GraphQL Fake API is ready to use 🚀
  Here are your links:

  ❯ Interactive Editor: http://localhost:9002/editor
  ❯ GraphQL API:        http://localhost:9002/graphql
```

**Comment by Antiarchitect** [comment]
Openssl log: https://gist.github.com/Antiarchitect/1ed9d8d9be67c12c5ec87cada0e8df6a
It seems like one of the root-ca inside the image has expired.

**Comment by IvanGoncharov** [comment]
Fixed in #118 
@Antiarchitect Big thanks for PR 👍
